modules/lambdas/main.tf

```diff
@@ -245,7 +245,7 @@ resource "aws_lambda_function" "internal_reconcile_report_orphan" {
   ## OPTIONAL
   description      = "Receives job id and page index from end user and returns reporting information of files that have records in the S3 bucket but are missing from ORCA catalog."
   filename         = "${path.module}/../../tasks/internal_reconcile_report_orphan/internal_reconcile_report_orphan.zip"
-  handler          = "internal_reconcile_report_orphan.handler"
+  handler          = "adapter_aws.handler"
```

Comment:
any particular reason to change the name?

Reply:
I opted to keep the business logic in the internal_reconcile_report_orphan.py, and move the AWS entry point (handler) to the AWS specific file.